astrosat_users/views/views_customers.py

```diff
@@ -271,7 +295,7 @@ class CustomerUserOnboardView(CustomerUserViewMixin, generics.GenericAPIView):
     A special view just for onboarding Users.
     """
 
-    permission_classes = [IsAuthenticated, IsAdminOrManager]
+    permission_classes = [IsAuthenticated, IsManagerPermission]
     serializer_class = CustomerUserSerializer
 
     def post(self, request, *args, **kwargs):
```

Comment:
Similarly, does this allow a manager to "onboard" (idk what that means exactly :) ) any CustomerUser?

Reply:
See above.